EssentialsDiscord/src/main/java/net/essentialsx/discord/JDADiscordService.java

```diff
@@ -156,7 +156,7 @@ public void startup() throws LoginException, InterruptedException {
 
         jda = JDABuilder.createDefault(plugin.getSettings().getBotToken())
                 .addEventListeners(new DiscordListener(this))
-                .enableCache(CacheFlag.EMOTE)
+                .enableCache(CacheFlag.EMOJI)
```

Comment:
Is that even needed?
createDefault does to my knowledge cache emojis by default, so this enable call is somewhat useless

Reply:
Want this explicit incase of any change in behavior that I miss during an update